static/js/browser2/util.ts

```diff
@@ -88,3 +88,21 @@ export function removeLoadingMessage(): void {
     loadingElem.style.display = "none";
   }
 }
+
+/**
+ * Returns the unit for a sourceSeries if there is a unit.
+ * @param sourceSeries
+ */
+export function getUnit(sourceSeries: SourceSeries): string {
```

Comment:
would be nice to start adding unittest for util functions

Reply:
done

Reply:
thanks!